index.html

```diff
@@ -309,6 +315,18 @@ <h3>Amol Agarwal</h3>
     window.addEventListener('DOMContentLoaded', (event) => {
       fetchDataFromFirebase();
     });
+    //The following list contains the colors used in the numbering and decoraion in the rows.
+    //[red, light red], [blue, light blue], [yellow, light yellow], [pink, light pink], [green, light green] colors respectively
+    var c = [["#fc3a3a", "#eb6363"], ["#5562d9", "#959de8"], ["#facc00", "#ffe77a"], ["#ff96ff", "#ffbdff"], ["#55b837", "#73d955"]];
+
+    //Following lists contain the elements with classes flex-box and special-border respectively
+    var fb = document.getElementsByClassName("flex-box");
```

Comment:
change the name from `flex-box` to `about-us-list-item`, flex box is a very general name and can it can interfere with something else.